### ISSUE TITLE
Please don't show a thick red banner for information

### ISSUE THREAD

**Issue by FlorianHeigl** [arXiv/html_feedback#3627]
**Describe the bug**

I can't read the text if my brain is telling me 10 times per second that there is a critical life threatening alert on my screen.
also submitting the issue doesn't seem to work.

**To Reproduce**
look at a paper

**Expected behavior**
a non-intrusive toned down colour is used.

**Screenshots**
 

<img width="1603" alt="Image" src="https://github.com/user-attachments/assets/4284f74f-ab53-4594-8da0-6438775776fa" />

**Desktop (please complete the following information):**
- OSX
- Chrome

**Additional context**

Opening a github issue via the presented button also doesn't work.

**Comment by chrisjcameron** [comment]
Hi @FlorianHeigl , thanks for the UI feedback. You raise a good point and while the red bar follows the rest of the site design, the html render is the only part of the site where we expect readers to spend significant time reading the content. I expect the feature will evolve over time. In the mean time, you might enjoy the unbranded html conversions available as part of the ar5iv labs integration: https://ar5iv.labs.arxiv.org 

Both project rely on the same underlying technology and we really appreciate these bug reports to help focus our development efforts as we improve the html conversion output. Thanks!

**Comment by dginev** [comment]
The page scaffold for HTML papers has been refreshed for new articles starting March 2026. 
The article reported in this issue (thank you!) has been updated to include that scaffold.

A full update of all arXiv HTML articles will follow at a later date.

You can use the new header buttons to hide/show the navigational menu, as well as the header itself. That choice will be saved for future visits in the same browser. The "Report Issue" button is now only available in the page header.

https://arxiv.org/html/2312.11871v3
